resources/puppetlabs/lein-ezbake/template/global/ext/cli/stop.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set +e
+
+pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
```

Comment:
we should make sure we pull in whatever supplies pgrep in the package dependencies. It should be there on most systems already, but it won't hurt.